server/src/config/test-config.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import 'reflect-metadata';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+
+import config from './server-config';
+
+let mongo: MongoMemoryServer;
+
+async function initDatabase() {
+  mongo = await MongoMemoryServer.create();
+  config.test.uri = await mongo.getUri();
+  return config.test;
+}
+
+async function closeDatabase() {
+  await mongo.stop();
+  await mongoose.disconnect();
+}
+
+export { initDatabase, closeDatabase };
```

Comment:
what's this file purpose ?

Reply:
I will add jsdoc but it for setup test and init and close database